components/GlobalButton/GlobalButton.styled.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+
+export const globalButton = styled.button`
+  background-color: ${(props) => {
+    switch (props.type) {
```

Comment:
all your "styled-components" props should begin with a `$`. Styled Components detects these props and don't forward them to the html elements. Otherwise, your html buttons get a "type" attribute in the html, which has real consequences on their behaviour. Therefore, please prefix all props you want to use exclusively in your styled components with `$`, e.g. `$type` and `$textColor`.